common/data/src/main/java/org/thingsboard/server/common/data/notification/template/DeliveryMethodNotificationTemplate.java

```diff
@@ -43,6 +44,7 @@ public abstract class DeliveryMethodNotificationTemplate {
 
     private boolean enabled;
     @NotEmpty
+    @NoXss
```

Comment:
Roll this back. Emails can have html. The rest, except web, don't need this.
But for web we cannot add it due to backward compatibility issues